docs/resources/aws_s3_bucket_object.md

```diff
@@ -0,0 +1,81 @@
+---
+title: About the aws_s3_bucket_object Resource
+---
+
+# aws_s3_bucket_object
+
+Use the `aws_s3_bucket_object` InSpec audit resource to test properties of a single AWS bucket object.
+
```

Comment:
Here, I think it might be worth explaining that each object has a 'key', which might be thought of as a path.